doc/configuration-flow.md

```diff
@@ -0,0 +1,54 @@
+# Overview of the configuration flow in Istio Manager
+
+Istio configuration consists of configuration objects. Each object has kind
+(for example, a routing rule), name and namespace uniquely identifying the
+object.  Istio Manager provides a REST-based interface for manipulating objects
```

Comment:
It is sort of the case, since `istioctl` essentially issues curl commands to kube API server. `curl` is difficult to show since we need to translate URL, add auth headers, and not do validation.